src/Frontend.hs

```diff
@@ -152,7 +151,7 @@ aulaMain =
   :<|> pure (Frame frameUserHack PageStaticTermsOfUse) -- FIXME: Generate header with menu when the user is logged in.
 
   :<|> Page.login
-  :<|> Page.logout
+  :<|> (redirect . absolutePath . relPath $ P.Login)
```

Comment:
While we are being pedantic, there is one more redirect in this file with a string literal path...